src/html/extractors/factories/embeddedAttrJs.ts

```diff
@@ -0,0 +1,31 @@
+import { JsParser } from '../../../js/parser';
+import { Validate } from '../../../utils/validate';
+import { Element, IHtmlExtractorFunction, Node } from '../../parser';
+/**
+ * embeddedAttrJsExtractor extractor messages from element attribute,
+ * but treat them as embedded js:
+ *
+ *     <span :title="_xn('ctx', 'msgid', 'plural', n, args)">See my title</span>
+ */
+
+export function embeddedAttrJsExtractor(selector: string, jsParser: JsParser): IHtmlExtractorFunction {
```

Comment:
You're not using `selector` at all. The current implementation just sends all attributes through the js parser.